packages/checkout/components/checkbox-control/index.tsx

```diff
@@ -44,23 +44,23 @@ const CheckboxControl = ( {
 				className
 			) }
 		>
+			<input
+				id={ checkboxId }
+				className="wc-block-components-checkbox__input"
+				type="checkbox"
+				onChange={ ( event ) => onChange( event.target.checked ) }
+				aria-invalid={ hasError === true }
+				{ ...rest }
+			/>
+			<svg
+				className="wc-block-components-checkbox__mark"
+				aria-hidden="true"
+				xmlns="http://www.w3.org/2000/svg"
+				viewBox="0 0 24 20"
+			>
+				<path d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z" />
+			</svg>
```

Comment:
This moves input and svg outside label and into div, now everyone is on the same level, but only text is inside label.